planning_interface/move_group_interface/src/move_group.cpp

```diff
@@ -689,6 +689,31 @@ class MoveGroup::MoveGroupImpl
     }
   }
 
+  bool validatePlan(const Plan &plan)
+  {
+    robot_state::RobotStatePtr current_state;
+    if (!getCurrentState(current_state))
+        return false;
+    if (plan.trajectory_.joint_trajectory.points.empty())
+        return true;
+
+    const trajectory_msgs::JointTrajectory &trajectory = plan.trajectory_.joint_trajectory;
+    const std::vector<double> &positions = trajectory.points.front().positions;
+    size_t n = trajectory.joint_names.size();
```

Comment:
moveit uses ``std::size_t`` thousands of times throughout its code base, so we should stick to that. i now see there is plenty of strong opinions on ``std::size_t vs size_t`` but I think if we were to switch you should do so in one big grep